data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/sink/TestSink.java

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package com.amazon.dataprepper.plugins.sink;
-
-
-import com.amazon.dataprepper.model.annotations.DataPrepperPlugin;
-import com.amazon.dataprepper.model.record.Record;
-import com.amazon.dataprepper.model.sink.Sink;
-
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.stream.Collectors;
-
-@DataPrepperPlugin(name = "test-sink", pluginType = Sink.class)
-public class TestSink implements Sink<Record<String>> {
```

Comment:
Why are we deleting the TestSink test cases? The TestSink.class still exists and is not removed in this PR. 

https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/sink/TestSink.java

Reply:
The TestPrepper and TestSource test classes are also marked for removal but the implemented classes still exist.

Reply:
Oh disregard, I have my paths mixed up